torch/_inductor/lowering.py

```diff
@@ -218,7 +218,36 @@ def in_namespace(op, namespace):
 
 
 def transform_args(args, broadcast, type_promotion_kind, convert_input_to_bool):
+    # 0-dim CPU tensor will be converted to a symbol when it is used in CUDA kernels
+    target_device_cuda = False
+    for arg in args:
+        if instance(arg.data, ir.BaseView) and arg.layout.device.type == "cuda":
```

Comment:
@eellison This is wrong. what's the correct way to check if a tensorbox has a layout?

Reply:
just call `get_device()`